### DISCUSSION TITLE
Is there any alternative to the BalloonTipClicked event?

### DISCUSSION THREAD

**Discussion by c0debeef** [HavenDV/H.NotifyIcon#100] [Q&A] [answered]
https://learn.microsoft.com/en-us/dotnet/api/system.windows.forms.notifyicon.balloontipclicked?view=windowsdesktop-7.0

**Comment by HavenDV** [comment, marked as answer]
TaskbarIcon.TrayBalloonTipShown/TrayBalloonTipClosed/TrayBalloonTipClicked routed events. It may not be available on some platforms

**Comment by c0debeef** [comment]
It's a pity that it can't bind a command directly.

```xaml
<i:Interaction.Triggers>
    <i:EventTrigger SourceName="xTray" EventName="TrayBalloonTipClicked" >
        <i:InvokeCommandAction Command="{Binding ShowMainWindowCommand}"/>
    </i:EventTrigger>
</i:Interaction.Triggers>
```
